src/redux/reducers/employer.ts

```diff
@@ -23,6 +23,8 @@ const initialState = {
     seniority: '',
     skills: [],
   },
+  jobLists: [], // this is store to list of all job poste created by the logged in employer
```

Comment:
We already have a `resource` state inside the store. It's better to save it in there:
`resource: { jobLists: [] }`

Reply:
resource have only skill sets, not for joblists

Reply:
The idea of `resource` is to store all the response data that will be used to display. So both `skillSets` and `jobLists` can be inside `resource`. Later we can put more stuff inside `resources` as well

Reply:
okay i see now there is a different reducer for resources already :-D